lib/rubocop/cop/infinum/factory_bot_association.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rubocop'
+
+module RuboCop
+  module Cop
+    module Infinum
+      # This cop looks for 'association' in factories that specify a ':factory' option
```

Comment:
ah, forgot about the description describing optimisation goals.

Reply:
Did not yet do it, was focusing on the big thingy 😄